server/db/schema/classes_taught.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS classes_taught (
+    class_id INT NOT NULL,
+    teacher_id INT NOT NULL,
+    
+    CONSTRAINT fk_class
+        FOREIGN KEY(class_id)
+        REFERENCES classes(id)
+        ON DELETE CASCADE
+    
+    CONSTRAINT fk_teacher
+        FOREIGN KEY(teacher_id)
+        REFERENCES teacher(id)
+        ON DELETE CASCADE
+)
```

Comment:
the class_id and teacher_id should be primary keys, could you add that change please?

Reply:
If I add `PRIMARY KEY` to `teacher_id`, then the following error will occur
![image](https://github.com/user-attachments/assets/b6594ca6-5279-4697-b812-2f85c2f5aba6)
(Screenshot is from a test environment on neon.tech)

Is there anyway else I can specify multiple `PRIMARY KEY`s for a table?

Reply:
Oh nevermind, I see how I am suppose to make the change.

Reply:
Are you talking about something like [this](https://www.w3schools.com/sql/sql_primarykey.ASP)?
![image](https://github.com/user-attachments/assets/fe035cd2-1d5c-43da-9b93-7beb98ead6c7)
I'm not sure if this will enforce Foreign Key since it looks like we have two columns made up by the same values.

Reply:
Yes that should work

Reply:
yup that looks good to me!